framework/Databases.py

```diff
@@ -212,15 +212,6 @@ def getInitParams(self):
     paramDict['exist'] = self.exist
     return paramDict
 
-  def getEndingGroupPaths(self):
-    """
-      Function to retrieve all the groups' paths of the ending groups
-      @ In, None
-      @ Out, histories, list, List of the ending groups' paths
-    """
-    histories = self.database.retrieveAllHistoryPaths()
-    return histories
-
```

Comment:
Was this code removed because unused?

Reply:
because it was not used and I do not envision to use it in the future.